### DISCUSSION TITLE
Filter Discussions by author

### DISCUSSION THREAD

**Discussion by royfrancis** [community/community#81350] [Discussions] [answered]
How to filter discussions by authors? 

![Screenshot 2023-12-21 at 14 25 54](https://github.com/community/community/assets/5083500/b0105b72-f43d-4628-a45b-1ba4f8adb2b2)

Issues has already got the UI right. Why is it all different when it comes to discussions?

![Screenshot 2023-12-21 at 14 32 44](https://github.com/community/community/assets/5083500/bce40273-79ad-4ccd-baa3-ef86ac22a293)

**Comment by harrylal** [comment, marked as answer]
Hi @royfrancis,To refine your search according to the author, you can employ the following format: "**author: [_author's name_]."** in search

![image](https://github.com/community/community/assets/42095702/f248922c-7eaf-48b3-b53e-c91159b79f2c)
